---
title: 'Supplementary text: Materials and Methods - Figures expression profiles (A case of insect female neoteny:
  E93 expression and links to the juvenile hormone in mealybugs)'
author: "Isabelle Vea"
date: "September 28, 2017"
output:
  word_document:
    fig_caption: yes
    keep_md: yes
  pdf_document: default
  html_document:
    keep_md: yes
---


#Summary
You will find here all the analyses related to the statistics of hormonal treatments carried out in Vea et al. (submitted to Journal of Insect Biochemistry and Molecular Biology). 


```{r}
library(ggplot2)
library(plyr)
library(tidyr)
```


#Effect of pyriproxyfen treatment
##Data
```{r}
#loading file JHM treatment
#females
dtf<-read.csv(file="E93-Pyr-N3D0.csv",header = TRUE)
names(dtf)
head(dtf)
dtf<-gather(dtf,Gene,SDM,10:14)
#dtf
head(dtf)

#males
dtm<-read.csv(file="E93-Pyr-male.csv",header = TRUE)
names(dtm)
head(dtm)
```

##Checking for data distribution 
```{r}
#female dataset subsetting to keep Kr-h1 and E93 1 and 2 subset from D4 to D8
dtf<-subset(dtf, Gene=="Kr.h1" | Gene=="PkE93.1"| Gene=="PkE93.2"| Gene=="PkE93.3")
dtf$Gene
dtf$DAT
```

###Comparing the absolute amount of reference gene between treatments
```{r}
dtfbox_RPL32 <- ggplot(dtf,aes(x=Treatment,y=rpL32)) + geom_boxplot(width=0.5, size=0.2, outlier.size = 0.1)

dtfbox_RPL32
hist(dtf$rpL32)
hist(log10(dtf$rpL32))

#test normality
shapiro.test(dtf$rpL32) ## not normal distribution
#non parametric test, Mann Whitney U test
wilcox.test(data=dtf, dtf$rpL32~dtf$Treatment, paired=FALSE) ## difference not significant

```





###Female dataset for PkKr-h1
```{r}
#all female dataset Kr-h1
dtfkrh1<-subset(dtf, Gene=="Kr.h1")
dtfkrh1$Gene
hist(dtfkrh1$SDM/dtfkrh1$rpL32, main="Distribution of all data  PkKr-h1/rpL32, regardless of treatment type")#not normal
```

The distribution of qPCR data collected for relative expression of PkKr-h1 is not normal. We decided to transform the data to log10.

```{r}
  #log transformed
hist(log10(dtfkrh1$SDM/dtfkrh1$rpL32)) #more or less normal
```

We also checked data distribution by treatment

```{r}
#by treatment
dtfkrh1CT<-subset(dtfkrh1,Treatment=="Control")
dtfkrh1CT$Treatment
dtfkrh1JHM<-subset(dtfkrh1,Treatment=="Mimic")
dtfkrh1JHM$Treatment

hist(dtfkrh1CT$SDM/dtfkrh1CT$rpL32, main="Distribution of PkKr-h1/rpL32 data, including only control treatments")
hist(dtfkrh1JHM$SDM/dtfkrh1JHM$rpL32, main="Distribution of PkKr-h1/rpL32 data, including only mimic treatments")

  #log transformed
hist(log10(dtfkrh1CT$SDM/dtfkrh1CT$rpL32)) #ok
hist(log10(dtfkrh1JHM$SDM/dtfkrh1JHM$rpL32)) #skewed normal
```




```{r}
#all female dataset E93 1

dtfE931<-subset(dtf, Gene=="PkE93.1")
dtfE931$Gene
hist(dtfE931$SDM/dtfE931$rpL32) #distribution not normal
```
```{r}
  #log transformed
hist(log10(dtfE931$SDM/dtfE931$rpL32))
```

```{r}
#by treatment
dtfE931CT<-subset(dtfE931,Treatment=="Control")
dtfE931CT$Treatment
dtfE931JHM<-subset(dtfE931,Treatment=="Mimic")
dtfE931JHM$Treatment

hist(dtfE931CT$SDM/dtfE931CT$rpL32) #not normal
hist(dtfE931JHM$SDM/dtfE931JHM$rpL32)
  
  #log transformed
hist(log10(dtfE931CT$SDM/dtfE931CT$rpL32)) #two peaks
hist(log10(dtfE931JHM$SDM/dtfE931JHM$rpL32)) #weird!

```


```{r}
#all female dataset E93 1
dtfE932<-subset(dtf, Gene=="PkE93.2")
dtfE932$Gene
hist(dtfE932$SDM/dtfE932$rpL32) #distribution not normal
  #log transformed
hist(log10(dtfE932$SDM/dtfE932$rpL32) )
```

```{r}
#by treatment
dtfE932CT<-subset(dtfE931,Treatment=="Control")
dtfE932CT$Treatment
dtfE932JHM<-subset(dtfE931,Treatment=="Mimic")
dtfE931JHM$Treatment

hist(dtfE931CT$SDM/dtfE931CT$rpL32) #not normal
hist(dtfE931JHM$SDM/dtfE931JHM$rpL32)
  
  #log transformed
hist(log10(dtfE931CT$SDM/dtfE931CT$rpL32)) #two peaks
hist(log10(dtfE931JHM$SDM/dtfE931JHM$rpL32)) #weird!

```









```{r}
#male dataset subset Krh1 and E93 1
head(dtm)
#subsetting Kr-h1 and E93 1
dtm<-subset(dtm, Gene=="Kr-h1" | Gene=="PkE93-1"| Gene=="PkE93-2"| Gene=="PkE93-3")
dtm$Gene
dtm$Day

#all male krh1
dtmkrh1<-subset(dtm, Gene=="Kr-h1")
dtmkrh1$Gene
names(dtmkrh1)
hist(dtmkrh1$SDM.Gene/dtmkrh1$SDM.rpL32) #not normally distributed
  
  #log transformed
hist(log10(dtmkrh1$SDM.Gene/dtmkrh1$SDM.rpL32)) #not normally distributed


#male Krh1 by treatment
#by treatment
dtmkrh1CT<-subset(dtmkrh1,treatment=="Control")
dtmkrh1CT$treatment
dtmkrh1JHM<-subset(dtmkrh1,treatment=="Mimic")
dtmkrh1JHM$treatment

names(dtmkrh1CT)
hist(dtmkrh1CT$SDM.Gene/dtmkrh1CT$SDM.rpL32)
hist(dtmkrh1JHM$SDM.Gene/dtmkrh1JHM$SDM.rpL32)

  #log transformed
hist(log(dtmkrh1CT$SDM.Gene/dtmkrh1CT$SDM.rpL32)) #ok
hist(log(dtmkrh1JHM$SDM.Gene/dtmkrh1JHM$SDM.rpL32)) #ok
```
We decided to use a log10 transformation of our absolute values normalized by the reference gene because the histograms did not show a normal distribution.


#statistical tests: general effect of JHM
## male on Kr-h1 expression by treatment and by day 
```{r}
par(mfrow=c(2,2))
#dtm
dtkrh1<-subset(dtm,Gene=="Kr-h1") #subsetting the dataset to have only Kr-h1
dtkrh1$Gene
names(dtkrh1)
is.numeric(dtkrh1$Day)
dtkrh1$treatment
names(dtkrh1)

#fitting linear model on transformed data
malelmkrh1<-lm(log10((dtkrh1$SDM.Gene/dtkrh1$SDM.rpL32))~dtkrh1$treatment+dtkrh1$Day+dtkrh1$treatment:dtkrh1$Day)
summary.lm(malelmkrh1) # there is an interaction between time and treatment
plot(malelmkrh1)
```

The treatment has a significant effect on PkKr-h1 expression (p-value 6.67e-5), gene expression doesn't seem to change significantly over time after treatment (0.4962), but an interaction between type of treatment and day after treatment was observed (0.0224). 
I therefore kept the interaction of variables in the model.


## male on E93-1 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtE931<-subset(dtm,Gene=="PkE93-1")
dtE931$Gene
dtE931$Day
is.numeric(dtE931$Day)
names(dtE931)

#transformed lm
malelmE931<-lm(log10((dtE931$SDM.Gene/dtE931$SDM.rpL32))~dtE931$treatment+dtE931$Day+dtE931$treatment:dtE931$Day)
summary.lm(malelmE931)  # no significant interaction
```
If we consider the linear model including an interaction between variables, the JHM treatment does not have a significant effect, but there is a change over time in gene expression. Because the interaction between these two variables are not significant, I removed it from the model considering only the two variables separately.

```{r}
#remove interaction
malelmE9312<-lm(log10((dtE931$SDM.Gene/dtE931$SDM.rpL32))~dtE931$treatment+dtE931$Day)
summary.lm(malelmE9312)
plot(malelmE931)
```
After removing the interaction, the JHM treatment has asignificant effect on gene expression and the expression changes over time in CT and JHM.


## male on E93-2 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtE932<-subset(dtm,Gene=="PkE93-2")
dtE932$Gene
dtE932$Day
is.numeric(dtE932$Day)
names(dtE932)

#transformed lm
malelmE932<-lm(log10((dtE932$SDM.Gene/dtE932$SDM.rpL32))~dtE932$treatment+dtE932$Day+dtE932$treatment:dtE932$Day)
summary.lm(malelmE932)
```

If we consider the linear model including an interaction between variables, the JHM treatment does not have a significant effect, but there is a change over time in gene expression. Because the interaction between these two variables are not significant, I removed it from the model considering only the two variables separately.


```{r}
#remove interaction
malelmE932<-lm(log10((dtE932$SDM.Gene/dtE932$SDM.rpL32))~dtE932$treatment+dtE932$Day)
summary.lm(malelmE932)
plot(malelmE932)
```
After removing the interaction, the JHM treatment has a significant effect on gene expression and the expression changes over time in CT and JHM.

## male on E93-3 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtE933<-subset(dtm,Gene=="PkE93-3")
dtE933$Gene
dtE933$Day
is.numeric(dtE933$Day)
names(dtE933)

#transformed lm
malelmE933<-lm(log10((dtE933$SDM.Gene/dtE933$SDM.rpL32))~dtE933$treatment+dtE933$Day+dtE933$treatment:dtE933$Day)
summary.lm(malelmE933)

```
If we consider the linear model including an interaction between variables, the JHM treatment does not have a significant effect, but there is a change over time in gene expression. Because the interaction between these two variables are not significant, I removed it from the model considering only the two variables separately.

```{r}

#remove interaction
malelmE933<-lm(log10((dtE933$SDM.Gene/dtE933$SDM.rpL32))~dtE933$treatment+dtE933$Day)
summary.lm(malelmE933)

plot(malelmE933)
```
After removing the interaction between variables, there is a significant effect of the treatment on E93 isoform 3 in males.

## Female on Kr-h1 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtfkrh1<-subset(dtf,Gene=="Kr.h1")
dtfkrh1$Gene
dtfkrh1$DAT
is.numeric(dtfkrh1$DAT)
names(dtfkrh1)

femalelmkrh1<-lm(log10((dtfkrh1$SDM/dtfkrh1$rpL32))~dtfkrh1$Treatment+dtfkrh1$DAT+dtfkrh1$Treatment:dtfkrh1$DAT)
summary.lm(femalelmkrh1)

```

When including the interaction between the two variables, there is a significant change in expression over time between the control and JHM groups. We therefore kept this model for PkKr-h1 in females.


## female on E93-1 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtf$Gene
dtfE931<-subset(dtf,Gene=="PkE93.1")
dtfE931$Gene
is.numeric(dtfE931$DAT)
names(dtfE931)

#with interaction
femalelmE931<-lm(log10((dtfE931$SDM/dtfE931$rpL32))~dtfE931$Treatment+dtfE931$DAT+dtfE931$Treatment:dtfE931$DAT)
summary.lm(femalelmE931) #no interaction

```
If we consider the linear model including an interaction between variables, the JHM treatment does not have a significant effect, but there is a change over time in gene expression. Because the interaction between these two variables are not significant, I removed it from the model considering only the two variables separately.


```{r}
#remove interaction
femalelmE931<-lm(log10((dtfE931$SDM/dtfE931$rpL32))~dtfE931$Treatment+dtfE931$DAT)
summary.lm(femalelmE931)
plot(femalelmE931)
```
There is a significant overall effect of the treatment on E93 isoform 1.


## female on E93-2 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtf$Gene
dtfE932<-subset(dtf,Gene=="PkE93.2")
dtfE932$Gene
is.numeric(dtfE932$DAT)
names(dtfE932)


femalelmE932<-lm(log10((dtfE932$SDM/dtfE932$rpL32))~dtfE932$Treatment+dtfE932$DAT+dtfE932$Treatment:dtfE932$DAT)
summary.lm(femalelmE932)# no effect of interaction

```
No significant effect when including the interaction between both variable, we therefore removed the interaction in the model.
```{r}
femalelmE932<-lm(log10((dtfE932$SDM/dtfE932$rpL32))~dtfE932$Treatment+dtfE932$DAT)
summary.lm(femalelmE932)
plot(femalelmE932)

```
The overall effect of treatment is significant, probably due to day 2, day 4 and Day 8

## female on E93-3 expression by treatment and by day
```{r}
par(mfrow=c(2,2))
dtf$Gene
dtfE933<-subset(dtf,Gene=="PkE93.3")
dtfE933$Gene
is.numeric(dtfE933$DAT)
names(dtfE933)
femalelmE933<-lm(log((dtfE933$SDM/dtfE933$rpL32))~dtfE933$Treatment+dtfE933$DAT+dtfE933$Treatment:dtfE933$DAT)
summary.lm(femalelmE933)
```
If we consider the linear model including an interaction between variables, the JHM treatment does not have a significant effect, but there is a change over time in gene expression. Because the interaction between these two variables are not significant, I removed it from the model considering only the two variables separately.
```{r}
femalelmE933<-lm(log((dtfE933$SDM/dtfE933$rpL32))~dtfE933$Treatment+dtfE933$DAT)
summary.lm(femalelmE933)
plot(femalelmE933)
```

Although E93 isoform 3 expression changes over time, there is no significant effect of the treatment.