---
title: 'Supplementary text: Materials and Methods - Figures expression profiles (A case of insect female neoteny:
  E93 expression and links to the juvenile hormone in mealybugs)'
author: "Isabelle Vea"
date: "November 30, 2018"
output:
  word_document:
    fig_caption: yes
    keep_md: yes
  pdf_document: default
  html_document:
    keep_md: yes
---

#Supplementary information 2.3. RNA extraction and quantitative RT-PCR
This document provides the R command lines used to produce the figures for E93 study in the Japanese mealybug. (REF)

#Needed packages
- ggplot2
- plyr
- tidyr

#Files

- sample2profile-Krh1E93.csv: qPCR data for rpL32, Kr-h1 and E93 isoforms for male and female expression profiles
- E93-Pyr-male.csv: qPCR data of JHM treatment on males
- E93-Pyr-N3D0.csv: qPCR data on JHM treatment on females


```{r}
#if packages not installed
library(ggplot2)
library(plyr)
library(tidyr)
```


#Expression profile of E93 during the post-oviposition development of male and female mealybugs
We first examined the expression profiles of PkKr-h1 and three identified isoforms of PkE93.
```{r}
#loading file expression profile
de<-read.csv(file="sample2profile-Krh1E93.csv",header = TRUE)
names(de)
```


This file includes the the second derivative maximum (SDM) values for PkKr-h1A, PkE93A, PkE93B, PkE93C and PkrpL32 used for expression profile analysis of males and females. The data was collected from RNA extraction performed on a new independent sampling from a previous study on juvenile hormone variations in the Japanese mealybug (Vea et al., 2016 PLOS ONE):


##PkKr-h1A expression between males and females (Figure S1)
```{r}
##PkKr-h1A expression profile (Figure S1)
FigS1<-ggplot(de,aes(x=Day.after.oviposition, y=de$Kr.h1/de$rpL32, fill=Sex)) + 
  geom_point(aes(shape=Sex)) + 
  geom_line(aes(linetype=Sex))+
  scale_shape_manual(values=c(1, 19))+ #shape of dots
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
  ylab("PkKr-h1A relative expression (/PkrpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  ggtitle("Figure S1: Expression profile of PkKr-h1A during\nmale and female Japanese mealybug development after oviposition")+
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))
FigS1
```

Our independent sampling showed a consistent expression pattern with the sampling carried out for Vea et al., 2016. PkKr-h1A expression shows a diverging expression when N2 start to differente between male and females. Male expression continues and drops later after the prepupal molt, while female N2 have their PkKr-h1 expression progressively decreasing, to remain unusually low until the adult molt.

##PkE93 isoform A (Figure 2A-top)
```{r}
##PkE93A expression profile (Figure 2A)
Fig2a1<-ggplot(de,aes(x=Day.after.oviposition, y=de$E93.1/de$rpL32, fill=Sex)) + 
  geom_point(aes(shape=Sex)) + 
  geom_line(aes(linetype=Sex) )+
  scale_shape_manual(values=c(1, 19))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
  ylab("PkE93A relative expression (/PkrpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  ggtitle("Figure 2A: Expression profile of PkE93A during\nmale and female Japanese mealybug development after oviposition")+
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))

Fig2a1
```  
PkE93A shows sexually dimorphic expression pattern starting from the middle of N2. PkE93A expression rapidly increases and peaks during the pupal stage, while in females, the levels are far lower, but with two small increases at the end of N2 and N3.


##PkE92 isoform B (Figure 2A-middle)
```{r}
##PkE93B expression profile (Figure S2)
Fig2a2<-ggplot(de,aes(x=Day.after.oviposition, y=de$E93.2/de$rpL32, fill=Sex)) + 
  geom_point(aes(shape=Sex)) + 
  geom_line(aes(linetype=Sex) )+
  scale_shape_manual(values=c(1, 19))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
  ylab("PkE93B relative expression (/PkrpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  ggtitle("Figure 2A middle: Expression profile of PkE93B during\nmale and female Japanese mealybug development after oviposition")+
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))

Fig2a2
```  
PkE93B shows similar patterns to PkE93A.

##PkE92 isoform C (Fig 2A-bottom)
```{r}
#PkE93C expression profile (Figure 2A)
Fig2a3<-ggplot(de,aes(x=Day.after.oviposition, y=de$E93.3/de$rpL32, fill=Sex)) + 
  geom_point(aes(shape=Sex)) + 
  geom_line(aes(linetype=Sex) )+
  scale_shape_manual(values=c(1, 19))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
  ylab("PkE93C relative expression (/PkrpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  ggtitle("Figure 2A: Expression profile of PkE93C during\nmale and female Japanese mealybug development after oviposition")+
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))

Fig2a3
```

PkE93C has generally very low expression pattern but is expressed during embryogenesis and the differential expression is observed sexually at the end of development too.


#Comparison PkKr-h1A and PkE93A (Figure 2B)
```{r}
head(de)
degather<-gather(de,Gene,SDM,7:10)
head(degather)
desub<-subset(degather, Gene=="Kr.h1"|Gene=="E93.1") #subset two genes
desub<-subset(desub, Day.after.oviposition >= 25) #just after N2

#with both male and female data 
Fig2B<-ggplot(desub,aes(x=Day.after.oviposition, y=(SDM/rpL32), fill=Gene)) + 
  geom_point(aes(shape=Gene), size=3) + 
  geom_line(aes(linetype=Gene) )+
  facet_wrap(~Sex, scales="free",nrow=1) +
  scale_shape_manual(values=c(3, 17))+
  scale_linetype_manual(values=c("twodash", "dotted"))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
ylab("Comparison relative expression PkE93A and PkKr-h1A in males (/rpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  ggtitle("Figure 2B: Comparison of PkKr-h1A and PkE93A\after N2 in males and females separately")+
 theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))

Fig2B
```


```{r}
#Fig 2B relative expression of PkE93A and PkKr-h1A in 100%
maleprepupa<-subset(de,de$Sex=="Male" & de$Day.after.oviposition >=27 &de$Day.after.oviposition <=35)

maleprepupa$E931_pct<-(maleprepupa$E93.1/maleprepupa$rpL32)/max(maleprepupa$E93.1/maleprepupa$rpL32)
maleprepupa$Krh1_pct<-(maleprepupa$Kr.h1/maleprepupa$rpL32)/max(maleprepupa$Kr.h1/maleprepupa$rpL32)

maleprepupagather<-gather(maleprepupa,Gene,SDM,7:12)

maleprepupa_pct<-subset(maleprepupagather, maleprepupagather$Gene=="E931_pct" |maleprepupagather$Gene=="Krh1_pct")

Fig2Bmale<-ggplot(maleprepupa_pct,aes(x=Day.after.oviposition, y=(SDM), fill=Gene)) + 
  geom_point(aes(shape=Gene), size=4) + 
  geom_line(aes(linetype=Gene) )+
  #facet_wrap(~Sex, scales="free",nrow=1) +
  scale_shape_manual(values=c(3, 17))+
  scale_linetype_manual(values=c("twodash", "dotted"))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
#ylab("Comparison relative expression PkE93A and PkKr-h1A in males (/rpL32)") +xlab("days after oviposition") +
  theme_classic(15) + 
  #ggtitle("Figure 2B: Comparison of PkKr-h1A and PkE93A\after N2 in males and females separately")+
 theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))


Fig2Bmale

```


#PkKr-h1A and PkE93B (Figure S2)
```{r}
#PkKr-h1 and PkE93B (Figure S2)
desub2<-subset(degather, Gene=="Kr.h1"|Gene=="E93.2") #subset two genes
desub2<-subset(desub2, Day.after.oviposition >= 25) #just after N2
#with both male and female data
FigS2<-ggplot(desub2,aes(x=Day.after.oviposition, y=(SDM/rpL32), fill=Gene)) + 
  geom_point(aes(shape=Gene)) + 
  geom_line(aes(linetype=Gene) )+
  facet_wrap(~Sex, scales="free",nrow=1) +
  scale_shape_manual(values=c(3, 17))+
  scale_linetype_manual(values=c("twodash", "dotted"))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
ylab("Relative expression (/rpL32)") +xlab("days after oviposition") +
    ggtitle("Figure S2: Comparison of PkKr-h1 and PkE93B\after N2 in males and females separately")+
  theme_classic(15) + 
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))
FigS2
```

#PkKr-h1A and PkE93-3 (Figure S3)
```{r}
#Kr-h1 and E93-3 (Figure S3)
desub3<-subset(degather, Gene=="Kr.h1"|Gene=="E93.3") #subset two genes
desub3<-subset(desub3, Day.after.oviposition >= 25) #just after N2

#with both male and female data
FigS3<-ggplot(desub3,aes(x=Day.after.oviposition, y=(SDM/rpL32), fill=Gene)) + 
  geom_point(aes(shape=Gene)) + 
  geom_line(aes(linetype=Gene) )+
  facet_wrap(~Sex, scales="free",nrow=1) +
  scale_shape_manual(values=c(3, 17))+
  scale_linetype_manual(values=c("twodash", "dotted"))+
  theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black"))+
ylab("Relative expression (/PkrpL32)") +xlab("days after oviposition") +
  theme_classic(15) +
     ggtitle("Figure S3: Comparison of PkKr-h1A and PkE93C\after N2 in males and females separately")+
  theme(axis.title.x = element_text(colour = "#242424",size=6), plot.title = element_text(size=8),axis.title.y = element_text(colour = "#242424",size=6))
FigS3
```



#JH mimic treatments
JHM treatments were performed on both male and female and we assessed their effect on development and gene expression (see 2.4.JHM assays on male prepupae and female juvenile instars for details on the treatments)

##Effect of pyriproxyfen treatment on gene expression: data
```{r}
#loading file JHM treatment
#females

dtf<-read.csv(file="E93-Pyr-N3D0.csv",header = TRUE)
names(dtf)
head(dtf)
dtf<-gather(dtf,Gene,SDM,10:14)
head(dtf)
dtf<-subset(dtf, Gene=="Kr.h1" | Gene=="PkE93.1"| Gene=="PkE93.2"| Gene=="PkE93.3")
dtf$Gene
dtf$DAT
```


```{r}
#males
dtm<-read.csv(file="E93-Pyr-male.csv",header = TRUE)
names(dtm)
head(dtm)
dtm<-subset(dtm, Gene=="Kr-h1" | Gene=="PkE93-1"| Gene=="PkE93-2"| Gene=="PkE93-3")
dtm$Gene
dtm$DayL
```

Plot rpl32 per treatments
```{r}
#female data
head(dtf)
rpl32f<-ggplot(dtf)+
geom_boxplot(aes(y=dtf$rpL32, x=dtf$Treatment))
rpl32f

#male data
head(dtm)
rpl32m<-ggplot(dtm)+
geom_boxplot(aes(y=dtm$SDM.rpL32, x=dtm$treatment))
rpl32m

```
The treatment did not affect rpl32 expression

Graphs of the effect of JHM treatments on Kr-h1 and E93 gene expression and statistical tests were obtained using the log10-transformed data (see statistical analysis file for more details [add a link to the file after github upload]).

```{r fig.width=7, fig.height=6}
#Fig 3A: male normalized data by PkrpL32 transformed by log 10
boxplot3A<-ggplot(dtm,aes(x=DayL,y=log10(SDM.Gene/SDM.rpL32))) + 
  geom_boxplot(aes(fill=treatment),width=0.5, size=0.2, outlier.shape=NA) + 
  geom_jitter(aes(color=treatment),position=position_jitterdodge(dodge.width=0.6),alpha=0.5,size=0.6) +
  facet_wrap(~Gene,scales="free",nrow=2) + 
  theme_classic(base_size = 12) +
  ggtitle("Figure 3A: Effect of JHM treatment on PkKr-h1A\and PkE93 expression in males transformed by log 10")+
  scale_fill_manual(name="", # Legend label, use darker colors
                  breaks=c("Control", "Mimic"),
                  labels=c("CT","5 mM"),
                  values=c("white", "grey")) +
  theme(strip.background=element_rect(fill="black")) +
  theme(strip.text=element_text(color="white",face="bold"))
  
  boxplot3A

```


```{r fig.width=7, fig.height=6}
#Fig 3B: females absolute values normalized by rpL32 and transformed by log10
dtflogbox <- ggplot(dtf,aes(x=DayL,y=log10(SDM/rpL32))) + 
  geom_boxplot(width=0.5, size=0.2, outlier.size = 0.1, aes(fill=Treatment)) + 
  geom_jitter(aes(color=Treatment),position=position_jitterdodge(dodge.width=0.6),alpha=0.5,size=0.6) +
  facet_wrap(~Gene, scales="free",nrow=2) + 
  theme_classic(base_size = 12) +
  scale_fill_manual(name="", # Legend label, use darker colors
                  breaks=c("Control", "Mimic"),
                  labels=c("CT","20 mM"),
                  values=c("white", "grey")) +
  ggtitle("Figure 3B: Effect of JHM treatment on PkKr-h1A and\ PkE93 expression in females transformed log10")+
  theme(strip.background=element_rect(fill="black"))+ 
  theme(strip.text=element_text(color="white", face="bold"))

dtflogbox
```
